packet.go

```diff
@@ -17,10 +18,23 @@ import (
 	"reflect"
 	"runtime/debug"
 	"strings"
+	"sync"
 	"syscall"
 	"time"
 )
 
+const maximumMTU = 1500
```

Comment:
wondering if this creates any limitation for packets in `lo` or jumboframes?

Reply:
that's actually the previous value only in a constant instead of having it hard coded :smile: